app/templates/suppliers/companies_house_number.html

```diff
@@ -37,7 +37,7 @@
             error = form.companies_house_number.errors[0],
             question_advice =
             'This is a unique 8-character number given to your organisation by Companies House.
-            \n\n<a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a>
+            Find your <a href="https://beta.companieshouse.gov.uk/">Companies House number</a>.
```

Comment:
Is this intentional, removing the `/n`s?  Those keep this sentence on a new line.